app/actions/remote/reactions.ts

```diff
@@ -7,8 +7,9 @@ import {addRecentReaction} from '@actions/local/reactions';
 import DatabaseManager from '@database/manager';
 import NetworkManager from '@init/network_manager';
 import {getRecentPostsInChannel, getRecentPostsInThread} from '@queries/servers/post';
-import {queryReaction} from '@queries/servers/reactions';
+import {queryReaction} from '@queries/servers/reaction';
```

Comment:
On this branch, I'm also seeing another file ...reaction**s**... should it be deleted?
 '@queries/servers/reactions';